code/game/objects/structures/trash_pile.dm

```diff
@@ -0,0 +1,102 @@
+/obj/structure/trash_pile
+	name = "trash pile"
+	desc = "A heap of garbage, but maybe there's something interesting inside?"
+	icon = 'icons/obj/trash_piles.dmi'
+	icon_state = "randompile"
+	density = TRUE
+	anchored = TRUE
+	layer = TABLE_LAYER
+	obj_flags = CAN_BE_HIT
+	pass_flags = LETPASSTHROW
+
+	max_integrity = 50
+
+	var/hide_item_time = 15
+
+	var/list/searchedby	= list()// Characters that have searched this trashpile, with values of searched time.
```

Comment:
put the comment on the line above and add an extra '/' so it is detected by auto documentation.

![image](https://user-images.githubusercontent.com/40974010/120123471-ebfbc180-c163-11eb-9913-bbb69d4bf2a6.png)
should look like this